### ISSUE TITLE
Can not Login using Third Party OAuth authentication 

### ISSUE THREAD

**Issue by ankur-sardar** [brave/browser-ios#1693]
### Description:

When Trying to Login in https://www.gitbook.com/login using `GitHub` account, it can not login to the website
### Steps to Reproduce

  1. Go to https://www.gitbook.com/login
  2. Sign In with GitHub
  3. Login Fails

**Actual result:**

**Expected result:**

**Reproduces how often:** [Easily reproduced/Intermittent issue]

**Brave Version:**

**Device details:**

**Website problems only:**
- did you check with Brave Shields down?

- did you check in Puffin/Dolphin (UIWebView-based browsers)? 

- did you check in Safari/Firefox (WkWebView-based browsers)?

**Comment by jhreis** [comment]
Did you try turning the shields off? Would help to isolate the issue.

**Comment by ankur-sardar** [comment]
@jhreis , Yes, Sheild is off, and still can not Redirect to the Previous Tab.
